ID.me WebVerify SDK/IDmeWebVerifyKeychainData.m

```diff
@@ -0,0 +1,106 @@
+//
+//  IDmeWebVerifyKeychainData.m
+//  WebVerifySample
+//
+//  Created by Mathias Claassen on 12/9/16.
```

Comment:
@mats-claassen We should be able to drop the "created by" since we're using git :)